include/utility/json_utility.hpp

```diff
@@ -382,13 +446,13 @@ inline bool procesSetGpioTag(const nlohmann::json& i_parsedConfigJson,
             l_pinValue);
         return true;
     }
-    catch (const std::exception& ex)
+    catch (const std::exception& l_ex)
     {
         std::string l_errMsg = "Exception on GPIO line: ";
         l_errMsg += l_pinName;
         l_errMsg += " Reason: ";
```

Comment:
Any change required/requested in the code which is outside the scope of the commit/PR should either be taken via a defect or only if in case it is breaking the existing code.
In case you both agree for this change, please take it in via an issue.

Reply:
created issue - https://github.com/ibm-openbmc/openpower-vpd-parser/issues/376 to update error message